rest-basic-example/src/main/java/be/c4j/security/soteria/TestServlet.java

```diff
@@ -37,14 +37,15 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
+
 package be.c4j.security.soteria;
 
 import java.io.IOException;
 
 import javax.annotation.security.DeclareRoles;
-import javax.security.authentication.mechanism.http.annotation.BasicAuthenticationMechanismDefinition;
-import javax.security.identitystore.annotation.Credentials;
-import javax.security.identitystore.annotation.EmbeddedIdentityStoreDefinition;
+import javax.security.enterprise.authentication.mechanism.http.BasicAuthenticationMechanismDefinition;
+import org.glassfish.soteria.identitystores.annotation.Credentials;
```

Comment:
I assume the change to use org.glassfish classes is because these classes didn't make it into the final spec?

Reply:
@bshannon Apparently, the samples were written long ago using initial version of security API. So I updated to use the latest version and fixed the package names accordingly.